src/Core/Geometry/MeshPrimitives.hpp

```diff
@@ -93,6 +93,13 @@ RA_CORE_API TriangleMesh makeCylinder( const Vector3& a,
                                        uint nFaces                                = 32,
                                        const Utils::optional<Utils::Color>& color = {} );
 
+RA_CORE_API TriangleMesh makeSharpCylinder( const Vector3& a,
```

Comment:
Not sure the name is adapted to what is built. The cylinder does not seems "sharp" as normals are smoothed radially. Add documentation to explain the difference between this and the existing MakeCylinder

Reply:
My idea is to remove the other makeCylinder, but i don't know if it's usefull for someone, any guess ?

Reply:
As your proposal covers also the previous makeCylinder (by setting `fillSegments` to 1), yes, your proposal should replace the previous one with the same name.

Reply:
Well the main difference is that the old one have the same normals for side and fill segment around sides. The new one have duplicated vertices with a sharp edge around side.